docs/site/downloads/cldr-48.md

```diff
@@ -164,6 +164,7 @@ The definition was unclear, and prone to misunderstanding — the more detailed
 #### ldmlBCP47
 - `type` — adds a new attribute `region`.
 - `keyboard3@conformsTo` — is updated to allow "48".
+- `hc` — adds values `c12` and `c24` as Technical Preview. Also see the note about `h24` in "V49 advance warnings". ([CLDR-18894][])
```

Comment:
Why do we sometimes write this as H24 and other times h24. Which is more accurate?